src/Configuration/NodeDefinition/DbNode.php

```diff
@@ -9,7 +9,19 @@
 
 class DbNode implements NodeDefinitionInterface
```

Comment:
```
... extends ArrayNodeDefintion

    public function __construct($name, NodeParentInterface $parent = null)
    {
        $this->create();
    }

    public function create(): void
    {
        $this->children() ... 
    }

```

Reply:
ten interface jsem dělal hlavně z toho důvodu aby se dal jednotně typehintovat v classách `*Definition` ale máš pravdu v tom že by šel hintovat přímo `NodeDefainition`

Reply:
Kluci, ja nechci bejt za kazisuka, ale to co tady delame je ve finale `$config = json_decode(file_get_contents('/data/config.json'))`

Reply:
@odinuv to máš pravdu, ale kam tím míříš? v tomhle jde jen o definici/validaci toho configu

Reply:
chtel jsem tim rict, ze nad tim onanujete :)

Reply:
@tomasfejfar káždopádně ještě před callem jsem to stihl upravit dle tvých připomínek